### DISCUSSION TITLE
where is Multi-Tenant (vendor)  object entity

### DISCUSSION THREAD

**Discussion by loveyandex** [vendurehq/vendure#1037] [Q&A]
we need accouinting and seller status 
what we can consider it

**Comment by skid** [comment]
AFAIK, there is no such thing. Vendure does not provide multi-vendor functionality out of the box. There is a suggestion to implement it here: https://github.com/vendure-ecommerce/vendure/discussions/895

**Reply by loveyandex** [reply]
@skid https://github.com/vendure-ecommerce/vendure/discussions/495 
READ THIS DISCCUTION 
THEY HAD A DISSCOTION WITH A TRICK CHANNEL TO SUPPORT MULTI SELLER
BUT THEY DID NOT PROIVE SELLER ENTITY IN DB and did not think all aspects of this functionallity
